imgui_demo.cpp

```diff
@@ -1891,7 +1891,7 @@ struct ExampleAppConsole
     {
         ClearLog();
         for (int i = 0; i < Items.Size; i++)
-            free(History[i]);
+            ImGui::MemFree(History[i]);
     }
```

Comment:
Nice catch too. I haven't seen it because I've done a %s/// from the one from the ClearLog actually because it is just after the Strdup that does the ImGui::MemAlloc! :)